### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by jboxxx** [Yuvaleros/material-ui-dropzone#266]
Hi - is there any handling for WCAG compliance?  All of the forms on the documentation fail to meet WCAG standard for not having an associated label.

I've tried using the common material-ui props such as label, id, aria-label, <InputLabel /> to create the label without luck.

`        <DropzoneArea
            id={"file-dropzone-area"}
            label={"Upload File"}
            acceptedFiles={['.csv']}
            onChange={(files) => handleDropzoneChange(files, handleDropzoneChangeCallback)}
            showFileNameInPreview={true}
            dropzoneText={'Drag and drop a csv of trades here, or click to choose a file!'}
            key={dropZoneKey}
          />`

The follow screenshot is from the accessibility checking tool called: WAVE

![Screen Shot 2020-12-20 at 11 04 12 AM](https://user-images.githubusercontent.com/15174662/102718000-b64cd580-42b3-11eb-908a-23ffd02dd3d4.png)

Thanks
